mne/preprocessing/ica.py

```diff
@@ -1200,7 +1200,7 @@ def find_bads_ecg(self, inst, ch_name=None, threshold=None, start=None,
             If True, data annotated as bad will be omitted. Defaults to True.
 
             .. versionadded:: 0.14.0
-        measure : {'zscore', "cor"}
+        measure : 'zscore' | 'cor'
```

Comment:
Not going to change this to "correlation"? I thought that was part of the plan?

Reply:
+1 to change to correlation with deprecation warning

Reply:
Done, without deprecation warning – has been introduced after 0.20